xdsl/dialects/experimental/stencil.py

```diff
@@ -131,6 +131,13 @@ def verify(self) -> None:
                 f"Expected 3 indexes for stencil.index, got {len(self.array.data)}."
             )
 
+    @staticmethod
+    def sizeFromBounds(lb: IndexAttr, ub: IndexAttr) -> Sequence[int]:
```

Comment:
I think the naming convention for python method and function names is `snake_case`. So this should be `size_from_bounds`.

Reply:
Will do